addons/website/controllers/form.py

```diff
@@ -252,6 +251,12 @@ def extract_data(self, model, values):
 
         return data
 
+    def _log_authenticate_message(self, record, message):
```

Comment:
This helper could be used to send an arbitrary message in the name of Odoo Bot, which seems strange. 

Maybe this should return a boolean to decide whether we want to log a message or not (It seems that the override is just skipping the call to super according to the model).

Reply:
+ FYI I think that using with_user(SUPERUSER_ID) will do the trick without having to force the author_id.

Reply:
Oh indeed, my bad! 

About using `with_user(SUPERUSER_ID)`, I am not against to use it instead of setting `author_id` if you think it is better to do that. 😁